larreco/RecoAlg/TrackMomentumCalculator.cxx

```diff
@@ -120,6 +183,103 @@ namespace {
     std::vector<double> const xmeas_;
     std::vector<double> const ymeas_;
     std::vector<double> const eymeas_;
+    double const correction_;
+  };
+
+  class FcnWrapperLLHD {
+  public:
+    explicit FcnWrapperLLHD(std::vector<double>& dEi,
+                            std::vector<double>& dEj,
+                            std::vector<double>& dthij,
+                            std::vector<double>& ind,
+                            std::vector<bool>& dthij_valid,
+                            double stepsize,
+                            double correction)
+      : dEi_{dEi}
+      , dEj_{dEj}
+      , dthij_{dthij}
+      , ind_{ind}
+      , dthij_valid_{dthij_valid}
```

Comment:
After switching the constructor arguments above to r-value references, then a move should be applied to each of them:

```c++
  : dEi_{std::move(dEi)}
  , dEj_{std::move(dEj)}
  , ...
```